packages/grafana-ui/src/components/Chart/Tooltip.tsx

```diff
@@ -68,3 +62,16 @@ export const Tooltip: React.FC<TooltipProps> = ({ content, position, offset }) =
 };
 
 Tooltip.displayName = 'ChartTooltip';
+
+const getStyles = (theme: GrafanaTheme) => {
+  return {
+    portal: css`
+      position: absolute;
+      top: ${theme.breakpoints.xs};
+      left: ${theme.breakpoints.xs};
```

Comment:
Not sure about this one, maybe just use `0` in this case.
```suggestion
      top: 0;
      left: 0;
```

Reply:
Alright 👍